spec/features/card_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe 'Reviewing card' do
+  subject! { create(:card) }
+
+  context 'When there are cards for review' do
+    before do
+      subject.update_attributes(review_date: Date.today - 2.days)
+    end
+
+    it 'card show on page' do
+      visit root_path
+      expect(page).to have_content(subject.translated_text)
+    end
+
+    context 'and submit correct answer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.